pages/mesosphere/dcos/1.14/administering-clusters/backup-and-restore/zookeeper/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: layout.pug
+navigationTitle: ZooKeeper State
+title: Back up and Restore ZooKeeper
+menuWeight: 1
+excerpt: Backing up and restoring DC/OS ZooKeeper state
+render: mustache
+model: /mesosphere/dcos/1.14/data.yml
+--- 
+
+Back up the state of ZooKeeper inside a DC/OS cluster, and later restore from that backup.
```

Comment:
This section describes the process of backing up and restoring the state of ZooKeeper running inside a DC/OS cluster.